sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md

```diff
@@ -21,6 +21,10 @@
 - Removed Model `UploadLogsError` representing the error and the associated logs that failed when uploading a subset of logs to Azure Monitor.
 - Removed Model `UploadLogsStatus` indicating the status of a logs upload operation.
 
+## 1.1.0 (2022-10-11)
+## Features Added
+- GA release with sovereign support
```

Comment:
Consider updating this to something like

Added multi-cloud support via `LogsIngestionAudience` to allow users to select the Azure cloud where the resource is located.